test/integration/node-specific/operation_examples.test.ts

```diff
@@ -1836,7 +1830,7 @@ describe('Operation (Promises)', function () {
             return collection.createIndex({ title: 1 }, { unique: true });
           })
           .then(function (indexName) {
-            test.ok(indexName);
+            expect(indexName).to.exist;
```

Comment:
Same with lint for empty function on line 1827.

Reply:
done